tests/10-client-api.spec.js

```diff
@@ -0,0 +1,158 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import dbHttpClient from '..';
+import isNode from 'detect-node';
+
+describe('http-client API', () => {
+  it('has proper exports', async () => {
+    should.exist(dbHttpClient);
+    dbHttpClient.should.have.keys([
+      'httpClient', 'ky', 'DEFAULT_HEADERS'
+    ]);
+    const {httpClient, ky} = dbHttpClient;
+    httpClient.should.be.a('function');
+    ky.should.be.a('function');
+  });
+  it('handles a network error', async () => {
+    const {httpClient} = dbHttpClient;
+    let err;
+    let response;
+    try {
+      // hit a non-existent endpoint on the karma server to avoid CORS concerns
```

Comment:
Not quite accurate when run on node.  Get ECONNREFUSED as there is no server.  Is that ok?  Also ok to assume no server happens to be running there?